setup.py

```diff
@@ -26,7 +26,7 @@
     python_requires=">=3.8",
     install_requires=["tutor<14"],
```

Comment:
It's correct here that we still require `tutor<14` (this patch does not include any Nutmeg-enabling changes), but we do also have to specify `tutor>=13.2.0` because that is when the v1 plugin API landed. See also https://github.com/overhangio/tutor/pull/599.

Reply:
Yes! Good catch! we need the lower bound. I'll fix it.